lib/manager/github-actions/extract.ts

```diff
@@ -7,7 +7,7 @@ import type { PackageDependency, PackageFile } from '../types';
 
 const dockerRe = regEx(/^\s+uses: docker:\/\/([^"]+)\s*$/);
 const actionRe = regEx(
-  /^\s+-?\s+?uses: (?<replaceString>(?<depName>[\w-]+\/[\w-]+)(?<path>\/.*)?@(?<currentValue>.+?)(?: # renovate: tag=(?<tag>.+?))?)\s*?$/
+  /^\s+-?\s+?uses: "?(?<replaceString>(?<depName>[\w-]+\/[\w-]+)(?<path>\/.*)?@(?<currentValue>.+?)(?: # renovate: tag=(?<tag>.+?))?)"?\s*?$/
```

Comment:
```suggestion
  /^\s+-?\s+?uses: "?(?<replaceString>(?<depName>[\w-]+\/[\w-]+)(?<path>\/.*)?@(?<currentValue>.+?)"?(?: # renovate: tag=(?<tag>.+?))?)\s*?$/
```

I don't think you want to wrap the comment?

Reply:
This should also support single quotes